pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -370,14 +368,14 @@ func (r *ReconcileWindowsMachineConfig) addWorkerNodes(count int) (int, []reason
 	return count, nil
 }
 
-// reasonError fulfils the error interface while also including a human readable reason for the error
-type reasonError interface {
+// ReconcileError fulfils the error interface while also including a human readable reason for the error
```

Comment:
Shouldn't this be changed to something else?

Reply:
Like what?